### ISSUE TITLE
Is this project still alive? Where is the documentation?

### ISSUE THREAD

**Issue by ghost** [codehaus/jettison#7]
I am looking for documentation, specifically the user documentation that the Apache CXF site directed here:http://jettison.codehaus.org/User%27s+Guide

I can see no userguide here at GitHub at all. Is this the official "Home" for the project?

**Comment by renannprado** [comment]
:+1:

**Comment by sberyozkin** [comment]
Please work with the https://github.com/jettison-json/ organization. I'll start working on a release there asap

**Comment by ronsigal** [comment]
Hi @sberyozkin,

Is  https://github.com/jettison-json/ the official reincarnation of jettison, then?

-Ron

**Comment by wangzhijiang00** [comment]
yeah, i use rest-easy , and i find the json-providers contains this project, but i can not find documention. so, i don't know is this project still alive..

**Comment by wangzj00** [comment]
thank you for your comments...thank you very much!!!

**Comment by tresf** [comment]
> Is https://github.com/jettison-json/ the official reincarnation of jettison, then?

It seems to be abandoned since 2018, no comments on issues or PRs for quite a while.  If there's a new home, please let people know so we can help contribute.

**Edit:** 2026 -- the above repo is alive, so I guess we use it!? 😅
